python/src/opendp/mod.py

```diff
@@ -798,6 +801,50 @@ def __hash__(self) -> int:
     def __iter__(self):
         raise ValueError("Metric does not support iteration")  # pragma: no cover
 
+class MultiDistance(Metric):
+    '''`MultiDistance` is a higher-order metric with multiple distance types for grouped data.'''
```

Comment:
```suggestion
    '''``MultiDistance`` is a higher-order metric with multiple distance types for grouped data.'''
```
Checked locally, and this renders as italics.

Just had a new idea about mixed markdown and rst. Filed:
- #2332